examples/AspNetCore/README.md

```diff
@@ -0,0 +1,68 @@
+# OpenTelemetry Example Application
+
+This project is an example of ASP.NET Core instrumentation for tracing and
+logging.
+
+The following exporters are configured for viewing the traces:
+
+* Console
+* Zipkin
+* Otlp
+* Jaeger
+
+The following exporters are configured for viewing the metrics:
+
+* Console
+* Otlp
+* Prometheus
+
+## Running the example
+
+Running instances of the following services are required to view exports:
+
+* Zipkin
+* OTEL Collector
+* Jaeger
+* Prometheus
+
+These can easily be spun up in docker containers.
+
+The project can be run from this directory as follows:
+
+```shell
+dotnet run
+```
+
+Instead of running the project and its dependencies individually, if you are
+using Docker Desktop, a `docker-compose` file is provided. This makes standing
+up the Zipkin, Otlp, Jaeger, and Prometheus dependencies easy, as well as
+starting the application.
+
+To run the example using `docker-compose`, run the following from this
+directory:
+
+```shell
+docker-compose up --build
+```
+
+With everything running:
+
+* [Invoke the API](http://localhost:5000/WeatherForecast) to create new data.
+* If you have run Zipkin with default settings:
+  * View your traces with Zipkin [here](http://localhost:9411/zipkin)
```

Comment:
Minor: consider replacing `here` with the actual URI (so the readers will see the URI / realize that it is a local endpoint).

```suggestion
  * View your traces with Zipkin by accessing the local endpoint [http://localhost:9411/zipkin](http://localhost:9411/zipkin).
```

Reply:
I have no problem with this change, but I was mirroring the convention in the microservice example.

Should I update the documentation for that here, or in a separate PR?

Reply:
I would suggest to keep this PR scoped - if we need to update documentation for another example, cover it in a separate PR.

Reply:
Sounds good. I've updated this per your recommendation, as well as the other instances in this README.